### DISCUSSION TITLE
Can't update to 0.7.8

### DISCUSSION THREAD

**Discussion by iuliuseq** [datenstrom/community#376] [Report a bug] [answered]
I can't update from 0.7.6 to 0.7.8 :( The Update plugin does not work in my web :( I cannot see the Update option in my Configurations

![blogiulius](https://user-images.githubusercontent.com/3447900/47138792-a17eec80-d2ba-11e8-892b-696c2ed8fca8.jpg)

**Comment by markseuffert** [comment]
Please click "About this website"... what do you see?

**Comment by iuliuseq** [comment]
I see this. Pressing Ok just do nothing.
![blogiulius2](https://user-images.githubusercontent.com/3447900/47139198-819bf880-d2bb-11e8-8c01-4dba1fdb4f9e.jpg)

**Comment by markseuffert** [comment, marked as answer]
Looks like your website has some old plugins that don't work with the automatic software update. Sorry about that. Here's how to fix it. Please delete file `system/plugins/setup.php`, then copy new plugins from a fresh installation into your `system/plugins` folder.

Let me know it this works?

**Comment by iuliuseq** [comment]
Thanks a lot. All seems to work fine right now. I have updated without problems :) I will update the gallery and slider plugins again and try them too

**Comment by markseuffert** [comment]
Thank you. Feedback and bug reports are always welcome. 😄
